index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
- [ ] [OPTIONAL] It is a good practice to always include `descriptive title` in our documents/webpage. Good titles on our webpages increase page SEO ranking and also allow screen reader users to access our website with ease. Please consider including `meaningful` title content that tells what your page is about.